aqua/input-leap/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        input-leap input-leap 198779cf9e798aac6ad41427e32d031305736de3
+version             2.4.0-20230416
+revision            0
+categories          aqua net sysutils
+platforms           darwin
+license             {GPL-2 OpenSSLException}
```

Comment:
for what is the `OpenSSLException`? Typically with the more liberal license open OpenSSL v3 I thought this isn't needed anymore, or?

Reply:
https://github.com/macports/macports-ports/pull/18251#discussion_r1173671028

I don't know anything about this one way or another.